web-ui/src/components/profile/Profile.jsx

```diff
@@ -1,24 +1,27 @@
-import React from "react";
+import React, { useContext } from "react";
+import ProfileContext from "../../context/ProfileContext";
 import "./Profile.css";
 
-const Profile = (profile = {}) => {
-const { image_url } = profile;
+const Profile = () => {
+  const context = useContext(ProfileContext);
+  const { bio, email, image_url, name, PDL, role } = context.defaultProfile;
```

Comment:
could you please change the variable `PDL` to be consistent with the others?
`const { bio, email, image_url, name, pdl, role } = context.defaultProfile;`